akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala

```diff
@@ -925,6 +925,10 @@ class ClusterSingletonManager(singletonProps: Props, terminationMessage: Any, se
         case None =>
           tryAcquireLease()
       }
+
+    case Event(HandOverDone, _) =>
+      // nothing to do
+      stay()
```

Comment:
The question is: is it a bug when we get a `HandOverDone` when we are in `Oldest` state - shouldn't we have been in `HandingOver`?

Or does it happen also for 'healthy' cases, for example perhaps we might get 2 `HandOverDone` messages in case a handover is retried?

Reply:
It was happening when I was doing rolling restarts of an akka cluster

Reply:
It was happening when I was doing rolling restarts of an akka cluster